docs/FunctionalTestBuildScripts.md

```diff
@@ -2,7 +2,7 @@
 
 This file records scripts used to compile the test files, in alphabetical order.
```

Comment:
>This [](http://example.com/codeflow?start=0&length=4)

Both CodeFlow and GitHub does not show all files for me... may be a max number of files limit?

this is the commit:

[https://github.com/microsoft/binskim/commit/547e02f2a9e6dd819400a3d7a49d7438bf0fc614](https://github.com/microsoft/binskim/commit/547e02f2a9e6dd819400a3d7a49d7438bf0fc614)

if you scroll to last file it is
"src/Test.UnitTests.BinaryParsers/PEBinary/PETests.cs"
but not showing here for me.